charts/em-anno/values.yaml

```diff
@@ -16,15 +16,15 @@ anno:
   livenessTimeout: 5
   livenessPeriod: 15
   livenessFailureThreshold: 10
+  aadIdentityName: em
   keyVaults:
-    rpa:
-      resourceGroup: rpa
-      secrets:
-        - AppInsightsInstrumentationKey
     s2s:
```

Comment:
unfortunately we also need to remove the s2s key so these should go to `em-anno`